src/pages/resources/migration/android/migrate-to-2x.md

```diff
@@ -0,0 +1,272 @@
+
+1. [Update dependencies](#update-dependencies)
+2. [Update SDK initialization](#update-sdk-initialization)
+3. [Update outdated API references](#update-outdated-api-references)
+
+### Update dependencies
+
+#### Gradle
+
+The Adobe SDK BOM artifact enables managing all compatible versions of Adobe Experience Platform Android extensions by specifying a single BOM version. This is now the recommended way to manage Android SDKs. If you are using Gradle to manage your app dependencies, the following example shows how to start using BOM in the `build.gradle` file.
+
+<InlineAlert variant="warning" slots="text"/>
+
+Using dynamic dependency versions is not recommended for production apps. Refer to this [page](../../../resources/manage-gradle-dependencies.md) for managing Gradle dependencies.
+
+<InlineAlert variant="warning" slots="text"/>
+
+Starting with version 2.0.0, the `sdk-core` bundle (which includes Core, Lifecycle, Identity, Signal) will no longer receive updates. You need to include those libraries individually as described below.
+
+```java
+dependencies {
+    //implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+    //implementation 'com.adobe.marketing.mobile:userprofile:1.+'
+
+    // Use Adobe SDK BOM artifact to manage all compatible versions of Adobe Experience Platform Android extensions.
+    implementation 'com.adobe.marketing.mobile:sdk-bom:2.+'
+    implementation 'com.adobe.marketing.mobile:core'
+    implementation 'com.adobe.marketing.mobile:identity'
+    implementation 'com.adobe.marketing.mobile:signal'
+    implementation 'com.adobe.marketing.mobile:lifecycle'
+    implementation 'com.adobe.marketing.mobile:userprofile'
+}
+```
+
+Save the `build.gradle` file and select `Sync Project with Gradle Files` in Android Studio to download the 2.x SDKs
+
+#### Manual library import
+
+If you are importing SDK libraries manually, make sure to update your libraries by downloading the most recent 2.x binaries directly from [Maven Central Repository](https://mvnrepository.com/artifact/com.adobe.marketing.mobile).
+
+### Update SDK initialization
+
+After you have imported the new Android libraries, you'll need to update SDK initialization code as described below. With Mobile Core version 2.0.0 and above, the `MobileCore.start()` API is no longer required. The SDK has simplified initialization and registration of extensions by providing the `MobileCore.registerExtensions()` API. After the given extensions have been registered, the SDK will be initialized and the completion block will be executed. The code which used to reside in the start() block will now reside in the `MobileCore.registerExtensions()`'s completion block.
+
+The following code snippets show the recommended initialization code for the 2.x Mobile SDKs.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Java
+
+<Tabs query="lang=java"/>
+
+Kotlin
+
+<Tabs query="lang=kotlin"/>
```

Comment:
No. I did not change current content w.r.t code samples.

Reply:
oh! I thought this was new code...
@calebk1 Can you make a note to update this whenever you get a chance using the `CodeBlock`? It will help clean up a lot of code clutter as we rely on Tabs heavily for different language variants.